cmd/agent/app/check_nojmx.go

```diff
@@ -0,0 +1,15 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-2019 Datadog, Inc.
+
+// +build !jmx
+
+package app
+
+import "fmt"
+
+// The Puppy Agent doesn't support JMX.
+func RunJmxListWithMetrics() error {
+	return fmt.Errorf("Not supported in the Puppy Agent.")
```

Comment:
Note: The error message `"Not supported in the Puppy Agent."` does not exactly match the condition checked (No JMX).
Is a more general message like "JMX is not supported" is acceptable?